### PR TITLE
PL-1057 Add deployment files and update ReadMe

### PR DESCRIPTION
**Pull request by heath-sureify**
Motivation
---
Utilize Platform>platform Jenkins job to deploy FeatureHub container, and update ReadMe to indicate this is a fork for Sureify's needs.

Modifications
---
Add docker-compose, Jenkinsfile, dev deploy YML, and application & database configuration files.

### CODE REVIEW

File: .deploy/dev.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+  #@data/values
+  ---
+  namespace: dev
+  name: featurehub
+  service:
+    name: featurehub
+    enabled: true
+    type: NodePort
```
Comment (heath-sureify) [inline .deploy/dev.yml:8]: Java container...what should be the "type"?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,48 @@
+def buildStatus = "FAILED"
+def project = "featurehub-install"
+def CLUSTER = ""
+def BRANCH = ""
+
+pipeline {
+    agent {
+        kubernetes {
+            inheritFrom 'node-16-npm-8'
```
Comment (heath-sureify) [inline Jenkinsfile:9]: Java container...what should be the "inheritFrom"?

**Comment by heath-sureify** [comment]
Crap...wrong repo. 👎

**Comment by rvowles** [comment]
If you need any help, let us know :-)

**Comment by heath-sureify** [comment]
> 

If you're actually able to delete this to remove the content of the files in the PR, that would be appreciated.

**Comment by rvowles** [comment]
I'm sorry, its not possible, I just search for how. However, you could reopen the same PR on your existing repo and then  force push an empty commit and its gone.

**Comment by heath-sureify** [comment]
Ok. I cannot push to the branch because I deleted it, and do not have rights to "Restore branch" on this PR. Are you able to do that?

**Comment by rvowles** [comment]
Sorry - no, I don't have access. I recommend you email Github support and try and encourage them to do it?

**Pull request closed without merging**